specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/security.json

```diff
@@ -1437,6 +1437,437 @@
                     }
                 }
             }
+        },
+        "/subscriptions/{subscriptionId}/providers/Microsoft.Security/northSouthHardenings": {
+            "get": {
+                "x-ms-examples": {
+                    "List all north-south hardening resources for a subscription": {
+                        "$ref": "./examples/NorthSouthHardenings/ListAllNorthSouthHardenings_example.json"
+                    }
+                },
+                "description": "Gets a list of north-south hardening resources for the subscription.",
+                "tags": [
+                    "NorthSouthHardenings"
+                ],
+                "operationId": "NorthSouthHardenings_ListAll",

```

Comment:
Naming convention for method that list resource in subscription level is `List` so operation id should be `NorthSouthHardenings_List`

Reply:
Done